ccd-definition/CaseField/CareSupervision/CMO-deprecated.json

```diff
@@ -1,50 +0,0 @@
-[
```

Comment:
assume that you checked that all fileds are not present in prod (https://tools.hmcts.net/jira/browse/RDO-9771)

Reply:
They are all transient fields so they shouldn't be